### ISSUE TITLE
Doesn't appear to work with Streamlit 1.33.0 using pages.

### ISSUE THREAD

**Issue by netnem** [cedricvlt/streamlit-condition-tree#5]
I stuck "example.py" inside of pages folder using latest 1.33.0

```
import streamlit as st
import pandas as pd
from streamlit_condition_tree import condition_tree, config_from_dataframe

# Initial dataframe
df = pd.DataFrame({
    'First Name': ['Georges', 'Alfred'],
    'Age': [45, 98],
    'Favorite Color': ['Green', 'Red'],
    'Like Tomatoes': [True, False]
})

# Basic field configuration from dataframe
config = config_from_dataframe(df)

# Condition tree
query_string = condition_tree(config)

# Filtered dataframe
df = df.query(query_string)

```

Output is entirely blank except for "None".  Tried "Build a query too" but also didn't work.

Using Microsoft Edge browser. Python 3.11

![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/32517635/9ee4fb26-a648-4d4d-a4de-bf2a83fec557)

**Comment by netnem** [comment]
Tried again on another PC - Firefox, Python 3.11.8 Win11. Same result:

I did find that "hovering" in certain parts did partially show the tree.

![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/32517635/43e0dd32-ad78-4132-93fd-8cb2e8e2f4fb)

**Comment by cedricvlt** [comment]
Hi @netnem,

This is the normal behaviour. The tree is invisible because it is empty. Rules will appear after you create them.
You can pass an argument _tree_ as input for defining a default tree, or set up a _placeholder_.

As for the buttons, they are only shown on hover by default. You can change it with the parameter _always_show_buttons_.

Does this resolve your issue ?

**Comment by netnem** [comment]
Yes, I think this makes sense now.  I had just copied and pasted the example and it looked completely broken.  

Even adding a placeholder didn't "look" right to me - and I know that's a subjective thing.

![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/32517635/16ed7277-b560-4e72-bd80-acc66b8f3f28)

vs what it looks like when a default rule is populated:
![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/32517635/f4545765-ff3d-4518-ba60-5a1abe4eb306)

Having that contrast between "black and grey" make its look correct.

Expectation:
![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/32517635/b08f80ba-2340-4581-8edb-d9bfc28b245d)

vs what I saw with the example:
![image](https://github.com/cedricvlt/streamlit-condition-tree/assets/32517635/a58442d7-b356-4604-97da-9a863caee804)

I do have a few suggestions:
- Include an "st.write(df)" in your example to display output. - also add "import streamlit as st"
- Add an option to left-align "add rule/Add Group - probably above the "Not"
- Maybe add always_show_buttons=True into the example.  


I do agree that I don't believe this is an issue. I think I just didn't quite understand the UI as it looked like pieces were missing.  I'm good to close this one.
